### PR TITLE
SKS: add new methods

### PR DESCRIPTION
**Pull request by falzm**
This change introduces new SKS-related methods.

### CODE REVIEW

**Comment by sgmac** [comment]
This looks very good :+1:   I don't think I could say that much honestly, I need to get familiar with `httpmock`. Thanks for adding me to the review.

**Review by sgmac — APPROVED** [review]
I tested your branch, all good. The CI passed tests, so there's not that much I can help. Feel free to add me to more reviews. Thanks.

**Comment by falzm** [comment]
Thanks for your feedback @sgmac ;)

File: sks.go [hunk 1]
```diff
@@ -212,6 +212,36 @@ func (c *SKSCluster) ScaleNodepool(ctx context.Context, np *SKSNodepool, nodes i
...
 
+// EvictNodepoolMembers evicts the specified members (identified by their Compute instance ID) from the
+// SKS cluster Nodepool.
+func (c *SKSCluster) EvictNodepoolMembers(ctx context.Context, np *SKSNodepool, members []string) error {
+	instances := make([]string, len(members))
+
+	for i := range members {
+		id := members[i]
+		instances[i] = id
+	}
```
Comment (illi-j) [inline sks.go:223]: I don't understand the goal of this part
why not use `members` directly?

**Review by pierre-emmanuelJ — APPROVED** [review]

File: sks.go (hunk 1 above)
Comment (falzm) [reply sks.go]: You're right, this isn't necessary – fixed in 30b5016.

**Review by illi-j — APPROVED** [review]

**Pull request merged**